repose-aggregator/components/filters/versioning-filter/src/test/java/org/openrepose/filters/versioning/util/ContentTransformerTest.java

```diff
@@ -48,6 +49,7 @@
  * @author jhopper
  */
 @RunWith(Enclosed.class)
+@SuppressWarnings("unchecked")
```

Comment:
The unmarshalling to a Map without any generics created a lot of warnings in IntelliJ.  I could have annotated each method, but the static initializer would have still been a warning that couldn't be suppressed except at the class level as far as I could see.